js/bitget.js

```diff
@@ -1344,9 +1341,17 @@ module.exports = class bitget extends Exchange {
         } else if (takerOrMaker === 'T') {
             takerOrMaker = 'taker';
         }
-        let side = this.safeString2 (trade, 'side', 'direction');
-        const type = this.parseOrderType (side);
-        side = this.parseOrderSide (side);
+        let type; // eslint-disable-line init-declarations
+        let side; // eslint-disable-line init-declarations
+        if (trade['type'] !== undefined) {
```

Comment:
This is a violation of this rule: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys